setup.py

```diff
@@ -36,7 +36,7 @@ def read_file(name):
     """,
     classifiers=[
         'License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)',
-        'Programming Language :: Python :: 2 :: Only',
+        'Programming Language :: Python :: 3 :: Only',
```

Comment:
siggen 1.0.0 is Python 3 only. Because of that, this effectively becomes Python 3 only. Python 2 is unsupported as of the end of this year and supporting Python 2 and 3 is a maintenance burden, so I'd rather we just stuck with Python 3.

If that turns out to be an unfeasible, we can figure something out.

Reply:
I expect any issues with python 3 only. This library is fairly simple and machines using this should have 3.